.jenkins

```diff
@@ -40,7 +40,7 @@ pipeline {
                         sh '''rm -rf build && mkdir -p build && cd build && \
                               /opt/cmake/bin/cmake \
                                 -DCMAKE_CXX_COMPILER=nvc++ \
-                                -DCMAKE_CXX_STANDARD=17 \
+                                -DCMAKE_CXX_STANDARD=20 \
```

Comment:
Do you mean to move to C++20 here, given that we will only move to C++17 in the (future) 4.0 release?

Reply:
This isn't terribly important and we might just drop it for this pull request but I think bumping the requested standard in the CI by one `c++14->c++17->c++20` makes sense in the end.